### ISSUE TITLE
UDP broadcast not support?

### ISSUE THREAD

**Issue by lesafe2017** [slackhq/nebula#208]
I know there are options in config.yml:
drop_local_broadcast: false
drop_multicast: false

but it still cannot forward udp broadcast packet through interface nebula1.
Here is the python script for test udp broadcast:
```
import socket
import sys
import time

message = 'very important data'
baddr = ('192.168.100.255', 5007)   # 255.255.255.255 is same

sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
sock.settimeout(0.2)
sock.bind(('192.168.100.11', 0))       #local nebula IP
sock.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)

try:
    while True:
        print 'sending', message
        sent = sock.sendto(message, baddr)
        time.sleep(2)

finally:
    print 'closing socket'
    sock.close()
```

On lighthouse vps:
```
tcpdump -n udp dst port 5007 -i nebula1
```
no packets can be captured.

It seems to be error of "unable to find host" at hostmap.go when queryVpnIP for broadcast IP.

Thanks for any help.

**Comment by nbrownus** [comment]
`nebula` does not currently handle these packets as all traffic is assumed to have a single destination. The configuration options are to drop these packets early instead of allowing nebula to attempt to discover the specific node at `192.168.100.255` in your example.

We may support some amount of mcast/bcast packets in the future.

**Comment by JohnMaguire** [comment]
Closed as not planned, per #504.
